charts/gitlab/Chart.yaml

```diff
@@ -11,3 +11,8 @@ keywords:
 - code review
 - wiki
 home: https://about.gitlab.com
+
+dependencies:
+  - name: common
```

Comment:
why this is needed?

Reply:
Let ingress use the right version in different versions of k8s